src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Remove Https configuration
+REM Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+netsh http delete sslcert ipport=0.0.0.0:8084
+call :CheckError %ERRORLEVEL% "WARNING: An error occurred while deleting sslcert from port 8084."
+
+netsh http delete sslcert ipport=0.0.0.0:44285
+call :CheckError %ERRORLEVEL% "WARNING: An error occurred while deleting sslcert from port 44285."
+
+netsh http delete sslcert ipport=0.0.0.0:443
+call :CheckError %ERRORLEVEL% "WARNING: An error occurred while deleting sslcert from port 443."
+
+goto :done
+
+:CheckError
+SET _ErrorLevel=%1
+if NOT [%_ErrorLevel%]==[0] if NOT [%_ErrorLevel%]==[1] (
```

Comment:
Means the rule does not exist. I should NOT have added it to the OpenFireWallPorts though. Will remove it.